applications/plugins/image/Kinect.h

```diff
@@ -502,6 +502,7 @@ class Kinect : public virtual core::objectmodel::BaseObject
 
     void draw(const core::visual::VisualParams* vparams)
     {
+#ifndef SOFA_NO_OPENGL
```

Comment:
just for my knowledge, why is this needed?
Is it linked with the multi-thread use?

Reply:
Hi @epernod !
No it is not directly linked to multihreading. The new feature with image data tyes in MultiThreading requires the image plugin. And compiling it I noticed this error with OpenGL in Kinect.h (when the lib FREENECT is found), so I just added this small fix, but perhaps I should have done it in another PR.

Reply:
no, it was just for understanding.